### PR TITLE
feat: add PoseReferenceChangeRBSTask

### PR DESCRIPTION
**Pull request by afonsofonbraga**

### CODE REVIEW

**Comment by afonsofonbraga** [comment]
@doudou

File: transforms.orogen [hunk 1]
```diff
@@ -36,6 +36,25 @@ task_context "PoseAndTwistFrameChangeRBSTask" do
     port_driven
 end
 
+task_context "PoseReferenceChangeRBSTask" do
+    needs_configuration
+
+    # The pose+velocity of the source frame w.r.t. the reference frame
```
Comment (doudou) [inline transforms.orogen:42]: only pose (same below)

Comment (doudou) [inline transforms.orogen]: only pose (same below)

**Review by doudou — APPROVED** [review]

**Comment by afonsofonbraga** [comment]
@doudou I don't have access to merge this PR. I think you should merge it.

**Pull request merged**